spectral-octal-number-issue.md

```diff
@@ -0,0 +1,109 @@
+# Non-Octal Numbers In Unquoted Strings
+
+## Issue Description
+
+When linting OpenAPI 3.x.x specifications, Spectral lint is not appropriately
+parsing strings per the YAML specification. This issue presents itself when
+strings that are made up of numbers that begin with the digit `0`, and contain
+the digits `8` and `9`, are not quoted in the OpenAPI specification.
+
+Per the [Integer Language-Independent Type for YAML Version 1.1](https://yaml.org/type/int.html),
```

Comment:
Maybe just "Per [YAML's `!!int` type](https://yaml.org/type/int.html)".

Reply:
Then again, I don't think you need to explain what an octal is. That might be perceived as a little condescending. Probably just point out that this should be a `!!str`, rather than an `!!int` as per the YAML specification.

Reply:
Updating.